src/components/client/campaigns/CampaignCard.tsx

```diff
@@ -57,6 +57,30 @@ const StyledCard = styled(Card)(({ theme }) => ({
     backgroundSize: 'contain',
     height: 200,
     transition: 'filter 0.3s, opacity 0.8s',
+    '@media(min-width: 320px)': {
+      height: 170,
+    },
+    '@media(min-width: 360px)': {
+      height: 200,
+    },
+    '@media(min-width: 375px)': {
+      height: 208,
+    },
+    '@media(min-width: 390px)': {
+      height: 217,
+    },
+    '@media(min-width: 414px)': {
+      height: 233,
+    },
+    '@media(min-width: 600px)': {
+      height: 220,
+    },
+    '@media(min-width: 900px)': {
+      height: 262,
+    },
+    '@media(min-width: 1200px)': {
+      height: 175,
+    },
```

Comment:
I think that we should fix the `minWidth` of the cards and then adjust the grid to keep cards relatively close to that `minWidth` so that the images stay in similar aspect ratio on most resolutions and then agree on some a height for the images themselves.

We should also probably communicate that to the people uploading the images, that they should upload the featured images in some specific aspect ratio.

Could you try that?

Reply:
> I think that we should fix the `minWidth` of the whole card and then adjust the grids that render the cards to keep them relatively close to that `minWidth` so that the images stay in similar aspect ratio on most resolutions. Then agree on some a height for the images themselves.

> We should also probably communicate that to the people uploading the images, that they should upload the featured images in some specific aspect ratio.

Overlooked some of MUI's capabilities when I posted the above suggestion, but perhaps the best approach here would be to just, add breakpoints for the number of columns to the grid layouts [link](https://mui.com/material-ui/react-grid/#responsive-values). This way we could maintain cards width easily.

As for the image's aspect ratio we could just, replace the `height:200` property  from `classes.media` with  `width: '100%`. Then we can pass aspectRatio property, to the [`<div> inside <CardMedia>` component](https://github.com/podkrepi-bg/frontend/blob/master/src/components/client/campaigns/CampaignCard.tsx#L132). This will automatically adjust the height of the img container, to the width of the card,  just so the set aspectRatio is maintained.

Reply:
You can also check the native CSS attribute `aspect-ratio`

https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio